src/scripts/pokemons/PokemonFactory.ts

```diff
@@ -50,6 +51,14 @@ class PokemonFactory {
             LogEvent('encountered shiny', 'shiny pokemon', 'wild encounter',
                 Math.floor(App.game.statistics.totalPokemonEncountered() / App.game.statistics.totalShinyPokemonEncountered()));
         }
+        if (roamer) {
+            Notifier.notify({
+                message: `🌠 You encountered a roamer ${name}! 🌠`,
```

Comment:
It's called "Roaming <pokemonName>". This change also needs to be applied a couple of other places.

Reply:
Sure, I can remove the emojis and change roamer -> roaming